acquia/acquia_cms
# ACMS-1399: Enable DAM on Site studio components

**Motivation**

Fixes #ACMS-1399[https://backlog.acquia.com/browse/ACMS-1399]

**Proposed changes**

Allow site studio components to select media from dam if dam module is enabled

**Alternatives considered**

NA

**Testing steps**


- Install site on this branch
- enable dam module and configure it
- Check that you are able to select media from dam if using site studio components

**Merge requirements**
- [_Enhancement_] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
